RadarSDK/RadarEvent.m

```diff
@@ -121,48 +121,50 @@ - (instancetype _Nullable)initWithObject:(id)object {
     if (typeObj && [typeObj isKindOfClass:[NSString class]]) {
         NSString *typeStr = (NSString *)typeObj;
 
+        // These strings should match the values (and order) of the server's event
+        // constants.
         if ([typeStr isEqualToString:@"user.entered_geofence"]) {
             type = RadarEventTypeUserEnteredGeofence;
         } else if ([typeStr isEqualToString:@"user.exited_geofence"]) {
             type = RadarEventTypeUserExitedGeofence;
+        } else if ([typeStr isEqualToString:@"user.dwelled_in_geofence"]) {
+            type = RadarEventTypeUserDwelledInGeofence;
         } else if ([typeStr isEqualToString:@"user.entered_place"]) {
             type = RadarEventTypeUserEnteredPlace;
         } else if ([typeStr isEqualToString:@"user.exited_place"]) {
             type = RadarEventTypeUserExitedPlace;
-        } else if ([typeStr isEqualToString:@"user.nearby_place_chain"]) {
-            type = RadarEventTypeUserNearbyPlaceChain;
         } else if ([typeStr isEqualToString:@"user.entered_region_country"]) {
             type = RadarEventTypeUserEnteredRegionCountry;
         } else if ([typeStr isEqualToString:@"user.exited_region_country"]) {
             type = RadarEventTypeUserExitedRegionCountry;
-        } else if ([typeStr isEqualToString:@"user.entered_region_state"]) {
-            type = RadarEventTypeUserEnteredRegionState;
-        } else if ([typeStr isEqualToString:@"user.exited_region_state"]) {
-            type = RadarEventTypeUserExitedRegionState;
         } else if ([typeStr isEqualToString:@"user.entered_region_dma"]) {
             type = RadarEventTypeUserEnteredRegionDMA;
         } else if ([typeStr isEqualToString:@"user.exited_region_dma"]) {
             type = RadarEventTypeUserExitedRegionDMA;
+        } else if ([typeStr isEqualToString:@"user.entered_region_state"]) {
+            type = RadarEventTypeUserEnteredRegionState;
+        } else if ([typeStr isEqualToString:@"user.exited_region_state"]) {
+            type = RadarEventTypeUserExitedRegionState;
+        } else if ([typeStr isEqualToString:@"user.entered_region_postal_code"]) {
+            type = RadarEventTypeUserEnteredRegionPostalCode;
+        } else if ([typeStr isEqualToString:@"user.exited_region_postal_code"]) {
+            type = RadarEventTypeUserExitedRegionPostalCode;
+        } else if ([typeStr isEqualToString:@"user.nearby_place_chain"]) {
```

Comment:
think we should keep this one grouped with the other place events given the context type

Reply:
I only moved it because that's where it is in the server's file.